src/core-sdk-tasks/Crossgen.cs

```diff
@@ -68,7 +68,7 @@ public override bool Execute()
                 // Copy both dll and pdb files to the destination folder
                 foreach(var file in files)
                 {
-                    File.Copy(file, $"{dest}{Path.GetFileName(file)}", overwrite: true);
+                    File.Copy(file, $"{dest}{Path.DirectorySeparatorChar}{Path.GetFileName(file)}", overwrite: true);
```

Comment:
Adding `Path.DirectorySeparatorChar` was not specific to Alpine, right?

Reply:
Yeah correct, that was a generic bug where `\temp\folder\file.dll` would get copied as `\dest\folderfile.dll` instead of `\dest\folder\file.dll`. This was causing other test failures around asset inventory.

Reply:
In general, `Path.Combine` should be preferred to better handle corner cases, e.g., when `dest` is `/`.  I think we can keep this code though.

Reply:
good point, fixed it now.